corehq/apps/domain/forms.py

```diff
@@ -389,8 +389,10 @@ def __init__(self, *args, **kwargs):
         self.can_use_custom_logo = kwargs.pop('can_use_custom_logo', False)
         super(DomainGlobalSettingsForm, self).__init__(*args, **kwargs)
         self.helper = hqcrispy.HQFormHelper(self)
-        self.helper[4] = twbscrispy.PrependedText('delete_logo', '')
-        self.helper[5] = twbscrispy.PrependedText('call_center_enabled', '')
+        self.helper[3] = twbscrispy.PrependedText('default_geocoder_location', '')
```

Comment:
Does this work regardless both when the user has `privileges.CLOUDCARE` and when they do not?

Reply:
reverted as this change is not needed. But it works in both cases.